ruleskit/ruleset.py

```diff
@@ -51,6 +51,9 @@ def __str__(self):
                 + [str(self[i]) for i in range(len(self) - RuleSet.NLINES, len(self))]
             )
 
+    def append(self, rule: Rule):
+        self.rules.append(rule)
```

Comment:
Add test of type of object Rule to avoid appending something else.

Reply:
good idea